master/static/migrations/20240306164812_remove-rm-name-expconf.tx.down.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE experiments
+SET config = config #- '{resource_manager}'
```

Comment:
this down migration doesn't make sense to me, can you explain it? i don't think the down migrations matter, but i'm curious.